CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Change log
 -----------
 
+* Drop the external initramfs and only use the initramfs bundled in the kernel [Florin]
```

Comment:
Can we merge this and have a follow-up PR that does the actual release?
This Pr is not for bumping the VERSION as a result of a release, it does not do the release, it just removes the old external initramfs.
I don't like the merge PR saying we merge the initramfs branch when it will also do a release